### PR TITLE
FINERACT-822 Enabling AssertEqualsArgumentOrderChecker check

### PR DESCRIPTION
**Pull request by percyashu**
FINERACT-822

### CODE REVIEW

**Comment by percyashu** [comment]
@xurror @awasum @vorburger LGTY?

File: fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AccountNumberPreferencesTest.java [hunk 1]
```diff
@@ -376,7 +376,7 @@ private void validateAccountNumberLengthAndStartsWithPrefix(final String account
             Assert.assertEquals(accountNumber.length(), prefix.length() + 9);
             Assert.assertTrue(accountNumber.startsWith(prefix));
         } else {
-            Assert.assertEquals(accountNumber.length(), 9);
+            Assert.assertEquals( 9,accountNumber.length());
```
Comment (xurror) [inline fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AccountNumberPreferencesTest.java]: Does this stuff also reject space separated commas. It's not a big deal I'd prefer if there is a space after the comma. Makes stuff more readable.

Comment (percyashu) [reply fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AccountNumberPreferencesTest.java]: Ok done that

**Review by xurror — APPROVED** [review]
This is way better :-)

File: fineract-provider/src/test/java/org/apache/fineract/notification/StorageTest.java [hunk 2]
```diff
@@ -126,7 +126,7 @@ public void testNotificationStorage() {
         verify(this.notificationGeneratorWritePlatformService, times(1)).create(refEq(notification, "createdAt"));
         verify(this.notificationMapperWritePlatformService, times(1)).create(refEq(notificationMapper, "createdAt"));
         verify(this.notificationGeneratorReadRepositoryWrapper, times(1)).findById(1L);
-        assertEquals(actualGeneratedNotificationId, Long.valueOf(1));
+        assertEquals(Long.valueOf(1),actualGeneratedNotificationId);
```
Comment (awasum) [inline fineract-provider/src/test/java/org/apache/fineract/notification/StorageTest.java:129]: Is there a need for a space here? I will just merge this.

**Review by awasum — APPROVED** [review]
LGTM

**Pull request merged**